packages/web-react/.eslintrc.js

```diff
@@ -25,7 +25,7 @@ module.exports = {
   settings: {
     'import/resolver': {
       node: {
-        extensions: ['.js', '.jsx', '.ts', '.tsx'],
+        extensions: ['.js', '.jsx', '.ts', '.tsx', '.md', 'raw', '.md?raw'],
```

Comment:
I tried to add this, but it didn't help. @literat please can you look into that? I ended up adding the global rule below.

Reply:
Ignoring this is the only option I know right now. So please remove the code above if it did not help.

Reply:
I have used and configured https://www.npmjs.com/package/vite-raw-plugin and it is fine solution, I think.